modules/build/src/test/scala/scala/build/tests/BuildTests.scala

```diff
@@ -897,4 +904,18 @@ abstract class BuildTests(server: Boolean) extends munit.FunSuite {
       expect(scalaCompilerJarNameOpt.contains("scala3-compiler_3-3.1.0.jar"))
     }
   }
+
+  test("Pure Java") {
+    val inputs = TestInputs(
+      os.rel / "Foo.java" ->
+        """
+          |""".stripMargin
```

Comment:
Do we want more tests for pure Java here?
Maybe smth with actual java code would be worth it?

Reply:
Oh yep, that's an oversight.